python/lsst/sconsUtils/dependencies.py

```diff
@@ -504,7 +504,6 @@ def _tryImport(self, name):
                 else:
                     module.config.addCustomTests(self.customTests)
                 return module
-        state.log.info("Failed to import configuration for optional package '%s'." % name)
```

Comment:
Yes, though ideally we'd want to hide it for required packages, to cut down on the redundant chatter.  That'd require a bit of refactoring, though, because at present `_tryImport` doesn't know whether its current target is required or not.